task-list/app/controllers/tasks_controller.rb

```diff
@@ -1,2 +1,11 @@
 class TasksController < ApplicationController
+  def index
+    @sorted_tasks = Task.list
+  end
+
+  def show
+    id = params[:id]
+    @task = Task.find(id)
+    @status = @task.status
```

Comment:
Do you need a separate instance variable for this or could you use `@task.status` within your view?